DocumentiPubblici/AnalisiDeiRequisiti/Sezioni/Requisiti/Funzionali.tex

```diff
@@ -133,27 +158,6 @@
     
     \rfun{O}{} & L'utente non autenticato può aggiungere prodotti nel carrello come ospite e, appena si autentica, vengono aggiunti al suo carrello personale. & Interna \\
     
-    \rfun{O}{} & L'amministratore può aggiungere account di venditori. & Capitolato \\
-    
-    \rfun{O}{} & L'amministratore può eliminare account di venditori. & Capitolato \\
-        
-    \rfun{O}{} & L'amministratore può fare il \glo{deploy} del sito. & Capitolato \\
-    
-    \rfun{O}{} & L'amministratore può aggiungere servizi di terze parti alla piattaforma. & Capitolato \\
-    
-    \rfun{O}{} & L'amministratore può rimuovere servizi di terze parti alla piattaforma. & Capitolato \\
-    
-    \rfun{O}{} & L'amministratore può specificare i requisiti minimi di una password. & Capitolato \\
-    
     \rfun{Z}{} & L'amministratore può modificare dinamicamente la copia del sito Web o cambiare il layout di PDP o PLP senza ridistribuire l'intero codice ad ogni modifica utilizzando \glo{\textit{Contentful}} come \glo{CMS}. & Capitolato \\
 
-    \rfun{O}{} & Nella schermata principale l'utente deve visualizzare i prodotti in evidenza. &  \\
-    
-    \rfun{O}{} & La ricerca di prodotti deve essere possibile da tutte le schermate. &  \\
```

Comment:
Questa sarebbe quella per la barra di ricerca, come mai l'hai tolta?

Reply:
o meglio sarebbe da modificare come nell'uc "ricerca i prodotti da plp o schermata principale"

Reply:
L'avevo tolto per sistemarlo in seguito quando faremo i requisiti

Reply:
L'ho messo tra commento così non ce lo perdiamo!